C/cryptography/Vernam_Cipher.c

```diff
@@ -0,0 +1,179 @@
+/*In Vernam cipher(also known as the One-Time Pad), the length of the plaintext, ciphertext, and key is the same.
+  Below is an implementation of Vernam cipher in the C language.*/
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#define LENGTH_MAX 100 //This defines the maximum length of the string in which we want to perform encrypt and decrypt operations.
+
+char *decrypt(char *ciphertext, char *key) { // This is the encrypt function which takes plaintext and key as input
+
+	static char decrypted[LENGTH_MAX];
+
+	for(int i = 0; i < strlen(key); i++) {
+
+		int ct = ciphertext[i] - 0;
+		ct -= 65;
+
+		int k = key[i] - 0;
+		k -= 65;
+
+		int pt = ct - k;
+		pt = pt % 26;
+
+		pt += 65;
+
+		char plaintext = pt;
+
+		decrypted[i] = plaintext;
+	}
+
+	return decrypted;		//Return the encrypted message	
+}
+
+char* encrypt(char *plaintext, char *key) { // This is the decrypt function which takes ciphertext and key as input
+
+	static char encrypted[LENGTH_MAX];
+
+	for(int i = 0; i < strlen(key); i++) {
+	
+		int pt = plaintext[i] - 0;
+		pt -= 65;
+
+		int k = key[i] - 0;
+		k -= 65;
+
+		int cipher = pt + k;
+		cipher = cipher %  26;
+
+		cipher += 65;
+
+		char enc = cipher;
+
+		encrypted[i] = enc;
+	}
+
+	return encrypted;               //Return the decrypted message
```

Comment:
Add comments to code segments. The comments should be above the segment and not alongside. 
Your code lacks comments. Please do the needful.